SettingsView/Platforms/Android/SettingsViewRecyclerAdapter.cs

```diff
@@ -176,18 +176,22 @@ public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int positi
 
             var vHolder = holder as ViewHolder;
             vHolder.RowInfo = rowInfo;
+
+            bool cellVisible = (rowInfo.Cell as CellBase)?.IsVisible ?? true;
             
             if(!rowInfo.Section.IsVisible ||
+               !cellVisible ||
                (rowInfo.ViewType == ViewType.CustomFooter && !rowInfo.Section.FooterVisible))
             {
                 vHolder.ItemView.Visibility = ViewStates.Gone;
                 vHolder.ItemView.SetMinimumHeight(0);
                 vHolder.ItemView.LayoutParameters.Height = 0;
                 return;
             }
-
+            
             vHolder.ItemView.Visibility = ViewStates.Visible;
-
+            vHolder.ItemView.LayoutParameters.Height = -2; //wrap_content
```

Comment:
Without this line, the parent section previously hidden and made visible again, was also hidden when hiding one of its cells.